js/meetup/data.js

```diff
@@ -1,4 +1,61 @@
 const meetups = [
+  {
+    episode: 22,
+    title: 'RoR Himalayan Meetup on Zoom',
+    date: '2023-01-17',
+    start_time: '05:00 pm',
+    end_time: '07:00 pm',
+    address: 'Remote Meetup',
+    hosted_by: {
+      name: 'Saroj Maharjan',
+      twitter_username: 'zoraslapen'
+    },
+    venue_sponsor: {},
+    sessions: [
+      {
+        id: 1,
+        title: 'Hanami 2.0',
+        description: null,
+        speaker: 'Tim Riley',
+        twitter_username: '',
+        github_username: 'timriley',
+        gitlab_username: '',
+        external_avatar: 'https://avatars.githubusercontent.com/u/40017038?v=4',
+        designation: 'Principal Engineer @ Buildkite',
+        slides_link: 'https://hanamirb.org/',
+        video_link: 'https://youtu.be/ZqbPUKMSSeg ',
+        demo_link: 'https://github.com/hanami/bookshelf'
+      },
+      {
+        id: 2,
+        title: 'Bonus: Dry-rb',
+        description: null,
+        speaker: 'Tim Riley',
+        twitter_username: '',
```

Comment:
Tim moved to mastodon https://ruby.social/@timriley from twitter https://twitter.com/timriley. But we don't link to mastodon currently

```suggestion
        twitter_username: 'timriley',
```

Reply:
I've added support for mastodon link